force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -126,33 +157,42 @@ public with sharing class ClassesObjectsMethods {
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
 
         // The variable to store the concatenated string
+        String concatName = '';
 
         // Loop through the list of scientists' names
-
-            // If the name doesn't contain the letter 'G', skip this iteration
-
-            // Add the name to the result string, followed by a comma
-
+        For (String name : scientistNames){
+            // if name with G is found then add, else skip
+            if (name.containsAny('gG')){
+               concatName += name + ',';
+            } else {
+                // If the name doesn't contain the letter 'G', skip this iteration
+                continue;
+            }
```

Comment:
This works but nothing happens after the continue so you should try continuing first then adding the value